apps/zotonic_mod_admin/src/support/z_admin_refers.erl

```diff
@@ -40,7 +40,7 @@ insert_ensure_refers_all_task(Context) ->
 
 %% @doc Check all resource for embedded resource references and add a refers
 %% edge for all of them.
--spec task_ensure_refers_all(FromId, Context) -> ok when
+-spec task_ensure_refers_all(FromId, Context) -> ok | {delay, integer(), list()} when
```

Comment:
Maybe it would be nice to also have a return type for pivot tasks, so that it could make these functions easily recognisable if not type-checkable.

If there were one, I suppose it would also be the return type of `z_pivot_rsc_task_job:call_function/4`

Reply:
Good one.

Adding the type `z_pivot_rsc:task_return/0`